install.rst

```diff
@@ -0,0 +1,106 @@
+Install an OMERO server
+=======================
+
+This chapter describes an installation of an OMERO.server and the most important post-installation steps, such as additional apps installation and LDAP configuration. The installation of the server on local machine is not recommended, except for testing or development purposes. The workflow is aimed at system administrators intending to familiarize themselves with OMERO installation.
+
+Description
+-----------
+
+We will show here:
+
+
+-  Where to find the OMERO.server requirements.
+
+-  How to install OMERO.server quickly using a simple Ansible playbook example.
+
+-  How to install OMERO.server very quickly using a provided Docker image.
+
+-  How to configure OMERO.server to work with LDAP, both using Ansible or using manual configuration.
+
+-  How to install apllications for OMERO.web, such as OMERO.figure.
+
+Resources
+---------
+
+Link to www OMERO at Institutions 
+Link to Simon's presentation (?)
+Link to Ansible
+Link to install docs
+Link to requrements
+Link to playbooks
+Link to App list
+
+Install a server
+----------------
+
+Get an empty box or virtual machine with CentOS 7.
+
+Ansible
+-------
+
+For OMERO installation, we recommend to use the Ansible software suite https://docs.ansible.com/ansible/latest/index.html which enables infrastructure as a code.
+
+Install ansible (at the time of writing, the ansible-core version was 2.11.12)::
+
+    $ pip install ansible
+
+Clone the deployment examples repository::
+
+    $ git clone https://github.com/ome/omero-deployment-examples.git
+    $ cd omero-deployment-examples
+
+Clone the subrepository for the one-node example::
+
+    $ git clone https://github.com/ome/ansible-example-omero-onenode.git
+    $ cd ansible-example-omero-onenode	
+
+Install requrements::
+
+    $ ansible-galaxy install -r requirements.yml
+
+Creat a file ``hosts.yml`` in the same directory and put into it (replacing the YOUR-HOST-NAME variable as appropriate)::
+
+    all:
+      hosts:
+        YOUR-HOST-NAME 
+
+Edit the ``playbook.yml`` file to change the OMERO root password (by default ``ChangeMe``). Then use this playbook to install the OMERO.server and OMERO.web on one node::
+
+    $ ansible-playbook --become -i hosts.yml playbook.yml
+
+Start OMERO.insight or a Command Line Interface (CLI) and log in to ``localhost`` with the username ``root`` and the password you have set in the ``playbook.yml`` above.
+
+Go to ``localhost/4080`` in your browser and log in to OMERO.web with the same credentials you used for OMERO.insight above.
```

Comment:
Should be: http://localhost:4080/ and probably nicer to make it into a clickable link